dev-aws/kafka-shared-msk/account-identity/holders.tf

```diff
@@ -0,0 +1,18 @@
+resource "kafka_topic" "account_identity_legacy_account_holder_events_compacted" {
```

Comment:
This topic was supposed to be created before mirroring was started

Reply:
is this gonna be an issue when applying?

Reply:
you have 2 options:
* option 1: start from scratch - not great, we'll need to drop all topics and consumer groups and start over
* option 2: import topics into tf state - more PRs but should be easy enough

Reply:
For option 2 you need to add an import block
```
import {
  to = kafka_topic.account_identity_legacy_account_holder_events_compacted
  id = "account-identity.legacy.account.holder.events.compacted"
}
```
and then remove it on a follow up PR

Reply:
Basically with this import statement we're telling Terraform that the resource already exists, and it shouldn't create it. It will still compare the current state in kafka against what we have declared in the .tf file, and apply the necessary changes if needed.

Another useful thing for you to consider, since I see there are [a lot of topics mirrored already](https://github.com/utilitywarehouse/kubernetes-manifests/pull/95161/files#diff-5754ddc8173df67d36bacb4f7883ec9a9ca1af9e1c2c1a96ea676394627d2c8aR23-R56), is to reverse engineer the configs from your source kafka cluster and then adjust. See this post for this: https://utilitywarehouse.slack.com/archives/C0536L8V7EU/p1716280603724679?thread_ts=1715870384.078699&cid=C0536L8V7EU

Reply:
@sbuliarca thanks, that's great

I updated this PR (fixed linting errors too) - to import this one topic for now - wanna see it working and then separately I will bring back all other topics (there's plenty of them). 

On the current config - the compression method is different than what's on kafka msk cluster - will this cause troubles?

Reply:
>On the current config - the compression method is different than what's on kafka msk cluster - will this cause troubles?

I don't expect to be any issues